### DISCUSSION TITLE
One more thing I would like to ask you that how to put something on asset

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#74] [Q&A]
I wanted to put txt file on assets and it would be read only txt file. So can you give me the code from which it can go to the txt file to see what's it's written. Please give me the code

**Comment by zdunex25** [comment]
https://docs.zepp.com/docs/1.0/reference/device-app-api/hmFS/open_asset/

`Page({
  build() {
    const fileId = hmFS.open_asset('test_file.txt', hmFS.O_RDONLY)
  }
})`

**Reply by alexmathew11** [reply]
Yeah but if I place the code it won't gonna work so give me detailed code

**Reply by zdunex25** [reply]
Have you tried debugging it? Believe me, console logs saved me hours of time when I could not figure out what I did wrong.

**Reply by alexmathew11** [reply]
Idk mate I don't know about debuging I just convert to an app
